R/setup.R

```diff
@@ -82,6 +82,22 @@ upload_enable <- as.logical(Sys.getenv(
 run_note <- as.character(
   Sys.getenv("WORKFLOW_RUN_NOTE", unset = get(params_obj_name)$run_note)
 )
+run_type <- as.character(
+  Sys.getenv("WORKFLOW_RUN_TYPE", unset = get(params_obj_name)$run_type)
+)
+
+# Must be one of the dedicated run types
+possible_run_types <- c(
+  "junk", "rejected", "test",
+  "baseline", "candidate", "final"
+)
+if (!run_type %in% possible_run_types) {
+  stop(
+    "Invalid run type '", run_type, "'. Must be one of: ",
+    paste0(possible_run_types, collapse = ", ")
+  )
+}
+rm(possible_run_types)
```

Comment:
Adding `run_type` handling so models get properly tagged on upload, same as the res model.